test-dir-o9Fqm9/test-errors.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Interesting that this directory got committed. It looks like one that is generated by the tests, but there must have been some issue that caused it not to be removed properly. Anyway, it should probably be deleted.

Reply:
Oops - I was moving too fast.  I'll delete.